core/src/main/java/org/apache/servicecomb/core/Invocation.java

```diff
@@ -185,18 +185,18 @@ public String getMicroserviceQualifiedName() {
     return operationMeta.getMicroserviceQualifiedName();
   }
 
-  public void triggerStartProcessingEvent() {
-    this.startProcessingTime = System.nanoTime();
-    EventUtils.triggerEvent(new InvocationStartProcessingEvent(
-        operationMeta.getMicroserviceQualifiedName(), this.invocationType));
+  public void triggerStartExecutionEvent() {
+    if (InvocationType.PRODUCER.equals(invocationType)) {
+      this.startExecutionTime = System.nanoTime();
+      EventUtils.triggerEvent(new InvocationStartExecutionEvent(operationMeta.getMicroserviceQualifiedName()));
```

Comment:
This EventUtils is more like a singleton.

Reply:
OK , delete EventUtils and make EventBus as a singleton